docs/index.md

```
@@ -121,68 +121,48 @@ discuss something, feel free to
-**No Impact**: If your configuration already uses `metadata.namespace == spec.namespace`, your setup will continue to work without changes.
+**Behavior change**: If you previously relied on `spec.namespace` pointing to a different namespace than `metadata.namespace`, the controller will now only search for Applications in the CR's own `metadata.namespace`.
 
 ### Examples
 
-#### Broken Configuration (Will Stop Working)
+#### Invalid Configuration (Will Be Rejected)
 
-This configuration will **stop working** because the CR is in `argocd` namespace but tries to update Applications in `production` namespace:
+This configuration will be **rejected** at admission because `spec.namespace` no longer exists in the API:
 
 ```yaml
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)